src/quickgui/qgsquickpositionkit.h

```diff
@@ -218,6 +226,9 @@ class QUICK_EXPORT QgsQuickPositionKit : public QObject
     //! \copydoc QgsQuickPositionKit::simulatePositionLongLatRad
     void simulatePositionLongLatRadChanged( QVector<double> simulatePositionLongLatRad );
 
+    //! \copydoc QgsQuickPositionKit::source
```

Comment:
How about "Emitted when the internal source of GPS location data has been replaced"